index.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  metadataStream: require('./src/components/metadataStream'),
+  extractFields: require('./src/components/extractFields').create,
```

Comment:
adding extractFields but not giving it the same testing treatment as metadata. Any reason for that?